AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -34,6 +36,13 @@ + (void)performDbTokenAsyncOperationWithHttpClient:(MSStorageIngestion *)httpCli
     NSData *payloadData = [NSJSONSerialization dataWithJSONObject:@{kMSPartitions : @[ partition ]} options:0 error:&jsonError];
 
     // Http call.
+    if ([MSAuthTokenContext sharedInstance].authToken) {
+      NSMutableDictionary *headers = [httpClient.httpHeaders mutableCopy];
```

Comment:
Is `httpHeaders` immutable?

Reply:
@jaelim-ms `httpHeaders` is `NSDictionary` that is immutable.